src/saml20_clj/xml.clj

```diff
@@ -30,6 +30,10 @@
     (.setFeature doc "http://xml.org/sax/features/external-general-entities" false)
     (.setFeature doc "http://xml.org/sax/features/external-parameter-entities" false)
     (.setFeature doc "http://apache.org/xml/features/nonvalidating/load-external-dtd" false)
+    (.setAttribute doc "http://www.oracle.com/xml/jaxp/properties/entityExpansionLimit" "2000")
+    (.setAttribute doc "http://www.oracle.com/xml/jaxp/properties/totalEntitySizeLimit" "100000")
+    (.setAttribute doc "http://www.oracle.com/xml/jaxp/properties/maxParameterEntitySizeLimit" "10000")
```

Comment:
The [example page](https://docs.oracle.com/javase/tutorial/jaxp/limits/using.html) uses `PARAMETER_ENTITY_SIZE_LIMIT`, without "max".

I'm not sure which is right, because Oracle is too cheap to host something useful like documentation at that URL.

The rest all seem to line up, at least.

Reply:
I double checked and it probably matches up: https://docs.oracle.com/javase/tutorial/jaxp/limits/limits.html

Reply:
Ahhh, that's the page I was looking for. Thanks!

Yep, looks good.